src/test/e2e/create-place.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { expect, test } from '@playwright/test';
+
+const randomUniqueUrl = `https://e2e-organizer-${Math.random()
+  .toString(36)
+  .substring(2, 7)}.com`;
```

Comment:
Would maybe add [faker](https://fakerjs.dev/) to the project since we'll likely need it in multiple E2E tests. We can also generate some `dummyPlace` attributes with it maybe

Reply:
Yes good suggestion 👍